### PR TITLE
Add project variants

### PR DESCRIPTION
**Pull request by joshlory**
Request from @poorvasingal.

### CODE REVIEW

File: apps/src/code-studio/initApp/project.js [hunk 1]
```diff
@@ -540,6 +540,12 @@ var projects = module.exports = {
       case 'studio':
         if (appOptions.level.useContractEditor) {
           return 'algebra_game';
+        } else if (appOptions.skin === 'iceage') {
+            return 'iceage';
+        } else if (appOptions.skin === 'infinity') {
+          return 'infinity';
+        } else if (appOptions.skin === 'gumball') {
+            return 'bumball';
```
Comment (ashercodeorg) [inline apps/src/code-studio/initApp/project.js]: Typo s/bumball/gumball.

**Comment by caleybrock** [comment]
cool! looks good, except for the typo asher pointed out

File: apps/src/code-studio/initApp/project.js [hunk 2]
```diff
@@ -540,6 +540,12 @@ var projects = module.exports = {
       case 'studio':
         if (appOptions.level.useContractEditor) {
           return 'algebra_game';
+        } else if (appOptions.skin === 'iceage') {
+            return 'iceage';
+        } else if (appOptions.skin === 'infinity') {
+          return 'infinity';
+        } else if (appOptions.skin === 'gumball') {
+            return 'gumball';
```
Comment (ashercodeorg) [inline apps/src/code-studio/initApp/project.js]: nit: Funny spacing.

Why isn't a linter catching this?

Comment (joshlory) [reply apps/src/code-studio/initApp/project.js]: Yuck 😕  I'm not sure...

Comment (joshlory) [reply apps/src/code-studio/initApp/project.js]: Looks like the `indent` ESLint rule [isn't enabled by default](http://eslint.org/docs/rules/) and [we don't intentionally enable it](https://github.com/code-dot-org/code-dot-org/blob/f0a35e985c3527ec7deb918829874dc7d75c1c8b/.eslintrc.js).

**Pull request merged**